clusterloader2/cmd/clusterloader.go

```diff
@@ -300,6 +301,11 @@ func main() {
 		klog.Exitf("Error while preloading images: %v", err)
 	}
 
+	if err := metadata.Dump(f, path.Join(clusterLoaderConfig.ReportDir, "cl2-metadata.json")); err != nil {
+		// Do not Exitf on metadata.Dump error which is not critical.
```

Comment:
I think the comment is redundant. The klog.Error is non critical as opposed to klog.Fatal